### PR TITLE
internal code cleanup

### PR DESCRIPTION
**Pull request by NotTheImpasta**
CheaterTim "Incredible Git Expert" no offense but your code is shit

### CODE REVIEW

**Comment by SquaredStudios-MC** [comment]
cant ask tim for one

**Review by SquaredStudios-MC — APPROVED** [review]
i wrote this code

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ If you want to add your project, you can create an issue on the issue tracker ta
 These are the types of properties available:
 * Links: Link, Discord, GitHub, Website, Docs, NPM, PyPI, Crates.io
 * Plain text: Version, Deprecated
-* Note that properties must be sorted in the order above
 
+Note that properties do not need to be in a spesific order.
```
Comment (CheaterTim) [inline README.md:17]: should be "specific"

File: src/util.ts [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+  const result = []
+  for(const key of Object.keys(obj).sort()) {
+    result.push(func(key, obj[key]))
+  }
+  return result
 }
 
-export function isMd() {
-    return matchMedia("(min-width: 768px)").matches
+export function isMobileDevice() {
```
Comment (CheaterTim) [inline src/util.ts:9]: it was called it `isMd` because it's equal to tailwind's `md:`

**Review by CheaterTim — CHANGES_REQUESTED** [review]
I didn't test the code

File: src/util.ts (hunk 2 above)
Comment (NotTheImpasta) [reply src/util.ts:9]: yeah but is mobile device is more readable

Comment (CheaterTim) [reply src/util.ts:9]: fair

Comment (RedVortexDev) [reply src/util.ts:9]: md is bigger than mobile iirc?

Comment (RedVortexDev) [reply src/util.ts:9]: yeah md: is medium screens and above so mobileDevice isnt correct, either change name or invert the return

Comment (SquaredStudios-MC) [reply src/util.ts:9]: Make your own pull request

**Pull request merged**